modules/setting/setting.go

```diff
@@ -776,15 +777,19 @@ func loadFromConf(allowEmpty bool, extraConfig string) {
 		PprofDataPath = filepath.Join(AppWorkPath, PprofDataPath)
 	}
 
+	LandingPageURL = LandingPageHome
 	switch sec.Key("LANDING_PAGE").MustString("home") {
 	case "explore":
 		LandingPageURL = LandingPageExplore
 	case "organizations":
 		LandingPageURL = LandingPageOrganizations
 	case "login":
 		LandingPageURL = LandingPageLogin
-	default:
-		LandingPageURL = LandingPageHome
+	case "custom":
+		customUrl := LandingPage(sec.Key("LANDING_PAGE_CUSTOM").MustString(""))
+		if customUrl != "" {
+			LandingPageURL = customUrl
```

Comment:
```suggestion
		customUrl := sec.Key("LANDING_PAGE_CUSTOM").MustString("")
		if customUrl != "" {
			LandingPageURL = LandingPage(customUrl)
```
(slightly more performant in an edge case even though it probably took longer to write compared to the time it saves...)